example/instruction_tuning_pipeline/finetune_clm.py

```diff
@@ -700,12 +704,7 @@ def concatenate_data(dataset, max_seq_length):
 
     if finetune_args.delta:
         if finetune_args.resume_peft != "":
-            if finetune_args.delta == "ssf":
-                peft_config = SSFConfig(target_modules=deltatuner_args.ssf_target_modules,
-                bias="none",
-                task_type="CAUSAL_LM",
-                )
-            model = DeltaTunerModel.from_pretrained(model, finetune_args.resume_peft, config=peft_config, denas_config=deltatuner_args)
```

Comment:
why we don't need SSFConfig now?

Reply:
does the SSFConfig class still will be used?

Reply:
> does the SSFConfig class still will be used?

Yes, it will.

> why we don't need SSFConfig now?

We need it, there are two ways to define `SSFConfig `, manually and automatically, here we add support for automatically definition (please refer to `e2eAIOK/deltatuner/deltatuner/deltatuner_model.py` and `e2eAIOK/deltatuner/deltatuner/tuner/deltatuner_ssf.py`), in this example, we choose to use automatically definition to simplify the example code, and we leave manually definition for advanced users.

@xuechendi  please have a check.